### ISSUE TITLE
♿ 🐞 Bug Report: WCAG attributes not well working as the should to be

### ISSUE THREAD

**Issue by macedvisioned** [elementor/elementor#21138]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The attributes in the progressbar in a website I checked on the website https://frmodulair.nl are not correct accessible.

I have tested it with default theme Astra and also in Hello on another website and in both themes the same issues.
In the website with Hello I have a minimum of plug-ins in use.

### Steps to reproduce

Case website with Astra theme
I set the following attributes in each progressbar, I have for bars:
In this case I set the same structure for set for 1 bar. 

role | progressbar
id | 65plussers2041
aria-label | Het aandeel van 65 plussers in Nederland in 2041
title | Het aandeel van 65 plussers in Nederland in 2041

Tested it with Axe Chrome plugin and got this message.
![WCAG-issue-Progressbar-theme-Astra](https://user-images.githubusercontent.com/2053468/214392108-6b865acb-40ff-40e3-8089-a5873ed2abcf.png)

Case website with Hello theme
I set the following attributes in each progressbar, I have for bars:
In this case I set the same structure for set for 1 bar. 

role | progressbar
id | Mijn vaardigheid
aria-label |  Mijn vaardigheid in toegankelijk webdesign
aria-labelledby | Toegankelijk design is voor iedereen belangrijk
title |  Mijn vaardigheid bepaald toegankelijk design voor iedereen 

Tested also the issue with the Hello theme:
![WCAG-issue-Progressbar-theme-Hello](https://user-images.githubusercontent.com/2053468/214392714-e5f43e5c-acf2-4a62-a24e-d7b3b6c389b7.png)

Nothing special seen in the widget code.
https://github.com/elementor/elementor/blob/master/includes/widgets/progress.php

In my opinion the issue is not designed in the Elementor element according the right WCAG rules.
https://dequeuniversity.com/rules/axe/4.6/aria-progressbar-name?application=AxeChrome

Please check this what could be wrong 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

systeminformation with Astra theme
[system-info-frmodulair.nl-24-01-2023.txt](https://github.com/elementor/elementor/files/10493344/system-info-frmodulair.nl-24-01-2023.txt)
systeminformation with only Hello theme with a minimum of plugins.
[system-info-www.edboon.nl-24-01-2023.txt](https://github.com/elementor/elementor/files/10493357/system-info-www.edboon.nl-24-01-2023.txt)

**Comment by nicholaszein** [comment]
Hello @macedvisioned! 

Thank you for submitting your issue! 🙏 

🏷️ This issue has been labeled and will be forwarded to the team for validation and possible solution. Please keep in mind that we can't provide any ETA on when this issue could be fixed if it is validated. We ask for patience.

Best regards

**Comment by macedvisioned** [comment]
@nicholaszein Hi Nicholas, can you give me an update of this issue?

**Comment by rami-elementor** [comment]
Hi @macedvisioned 

We fixed this issue in v3.13, you can read about it in our [developers update blog post](https://developers.elementor.com/elementor-3-13-developers-update/).

**Comment by macedvisioned** [comment]
@rami-elementor nice to read. Will check this improvements

**Comment by macedvisioned** [comment]
@rami-elementor I don't thinks this can be closed. Just testing, yesterday still issues in combination with containers. I will let you know a.s.a.p.

**Comment by rami-elementor** [comment]
Ref: https://github.com/elementor/elementor/pull/22031/files
